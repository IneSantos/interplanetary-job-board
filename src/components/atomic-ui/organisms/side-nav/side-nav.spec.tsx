import React from "react";
import { render } from '@testing-library/react';
import SideNav from "./side-nav";

describe("SideNav atom tests", () => {
  test(`SideNav component render with no lists`, () => {
    const { container } = render(
      <SideNav clearFilters={jest.fn()} accordionOptions={[]} searchInput="This is a search input" handleSearch={jest.fn()}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
