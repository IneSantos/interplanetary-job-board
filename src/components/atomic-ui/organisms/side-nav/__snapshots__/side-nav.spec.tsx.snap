// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNav atom tests SideNav component render with no list 1`] = `
<div
  class="side-nav "
>
  <div
    class="hamburger "
  >
    <span
      class="bar"
    />
    <span
      class="bar"
    />
    <span
      class="bar"
    />
  </div>
  <div
    class="side-nav-content"
  >
    <div
      class="filter-section"
    >
      <h3>
        What do you want to do?
      </h3>
      <a>
        Clear Filters
      </a>
      <input
        class="job-term"
        placeholder="Software engineering, Design, Sales"
        type="text"
        value="This is a search input"
      />
    </div>
  </div>
</div>
`;
